docs/GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,190 @@
+First, create a directory e.g: `~/images/`.
+Then, inside this directory, copy the artifacts you need to update:
+
+```shell-script
+$ cp /path/to/kogito-apps/jobs-service/target/jobs-service-8.0.0-SNAPSHOT-runner.jar .
+```
+
+Also, inside this directory, you can have a `Dockerfile` which can be used to update the artifact.
+
+```shell-script
+$ vim jobs-service-Dockerfile
+``` 
+
+```Dockerfile
+FROM quay.io/kiegroup/kogito-jobs-service
+COPY jobs-service-8.0.0-SNAPSHOT-runner.jar ${KOGITO_HOME}/bin/kogito-jobs-service-runner.jar
+```
+**Note**: Usually, latest artifacts are with version  `8.0.0-SNAPSHOT`
+**Note**: You can check the location of the artifact in the image from the [kogito-images](https://github.com/kiegroup/kogito-images) repository.
+
+Now you are going to build the image. Inside the above created directory, run:
+
+```shell-script
+$ podman build -t <prefered-tag-name>  -f jobs-service-Dockerfile  .
+```
+
+The above command will build your image with `quay.io/<your-namespace>/<image-name>` and can be seen in the output of `$ podman images`
+
+**Note**: We added the `-f` flag to specify the `Dockerfile` in case you want to build multiple images, you can then have different files and specify them with the `-f` flag.
+This image needs to be on a public container registry from which it can be pulled later.
+```shell-script
+$ podman push quay.io/<your-namespace>/<image-name>
+```
+
+## Build the Kogito Service Image
+
+### Base Images
+  * kogito-quarkus-ubi8
+  * kogito-quarkus-jvm-ubi8
+  * kogito-springboot-ubi8
+
+As an example for building the Image for your Kogito custom Service we’re going to build the example  `process-business-rules-quarkus` from [kogito-examples](https://github.com/kiegroup/kogito-examples) repository.
+
+```shell-script
+$ git clone https://github.com/kiegroup/kogito-examples
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`. With the following content:
+
+```Dockerfile
+FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+Now let’s push the above built image to a public container registry.
+
+```shell-script
+$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest
+```
+
+## Install the Operator
+
+First we would need to enable the OLM in our Minikube cluster. For that, just run:
+
+```shell-script
+$ minikube addons enable olm
+```
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
```

Comment:
do I need go for this to work? I have cloned and run the `make run-console-local` successfully, although it did complain about go.

Reply:
Ah yes, the project requires the `go1.12+` to be installed, I'll update that in per-requisites.